xradar/src/main/java/com/orzangleli/radar/TextPaint.java

```diff
@@ -0,0 +1,58 @@
+package com.orzangleli.radar;
+
+import ohos.agp.render.Paint;
+
+public class TextPaint extends Paint {
```

Comment:
Remove unused declarations in this and other classes.

Reply:
In TextPaint class not a single method is used, So can I remove this class and directly use super class(i.e. Paint) ?

Reply:
Remove TextPaint class.